vizro-core/tests/unit/vizro/models/test_page.py

```diff
@@ -95,3 +96,12 @@ def test_action_auto_generation_valid(self, standard_px_chart):
         assert len(page.actions) == 1
         assert isinstance(page.actions[0], ActionsChain)
         assert page.actions[0].id == f"{ON_PAGE_LOAD_ACTION_PREFIX}_Page 1"
+
+
+# TODO: Add unit tests for private methods in page build
+class TestPageBuild:
+    def test_no_left_side_container(self):
+        dashboard_title, nav_panel, control_panel = None, None, None
+        left_side_elements = [dashboard_title, nav_panel, control_panel]
+        left_side = html.Div(children=left_side_elements, className="left_side") if any(left_side_elements) else None
+        assert left_side is None
```

Comment:
This test doesn't evaluate the `Page.build` method and will not produce a failure even if I undo the changes made in` _arrange_containers`. In fact, it remains unaffected regardless of any modifications made to the Page or Vizro code. Consequently, it doesn't contribute to code coverage improvement and should likely be replaced with a test that test the Page.build method directly.

Reply:
Yes, exactly that's why I added a TODO comment in there that we still have to properly test the page.build and already created a ticket for it here: https://github.com/McK-Internal/vizx-core/issues/1084

This has to be properly tested though, so including all of the private methods in the page.build. So this is just a simple test for the simple if/else as the private methods of the page.build have to be properly tested in another PR which is planned for next sprint.

Reply:
Thanks for removing the test.